system/lib/pthread/pthread_testcancel.c

```diff
@@ -8,15 +8,11 @@
 #include "pthread_impl.h"
 #include <pthread.h>
 
-int _pthread_isduecanceled(struct pthread* pthread_ptr) {
```

Comment:
Sorry to be a pain but can you make the PR *only* about the removal of this function.. I really like simple PRs that do just one thing.

Reply:
No problem, I thought this simplification was too small for a dedicated PR, but maybe that doesn't matter. Reverted with commit 1bb8198bab97b47d1b36d15d8184dcb73f095ba9.